packagelist/debian_12_headless.txt

```diff
@@ -73,10 +83,24 @@ autoconf
 automake
 build-essential
 command-not-found
+dialog
 gdb
 git
+initramfs-tools
+libgpiod-dev
 libiio-dev
+libiio0-doc
```

Comment:
I started including this one because there have always been discrepancies between the IIO documentation online and what is packaged in our Linux distributions.

Reply:
Is there any chance we didn't look at the iio documentation matching the libiio used by a given debian release?  I haven't been burned by documentation differences yet. They have a list of documentation + versions here:
https://analogdevicesinc.github.io/libiio/
I'd like to see this come up with a real example we cannot otherwise solve before include documentation in our rootfs.

Reply:
Looking up the version and going back in the online docs are certainly another way of doing it. However, I have encountered cases where the person was looking at the latest, not knowing the documentation for the version in their distro is authoritative over (and different from) the latest.

Precedent? Pfft!-) We already include documentation in our rootfs, and not only man pages (which are _de-rigeur_ on a UNIX development system):
```
root@tsimx6ul:~# find /usr/ -name '*.html' | wc
     30      30    1347
root@tsimx6ul:~# 
```

The 71x0 boards have never shied away from having documentation on their rootfs, and getting the *right* docs for the installed version is really nice, especially as things change over time.

Reply:
For bringing along the 71x0s into distro-seed, it seems the question should come down to: what if any disadvantages are there to keeping them consistent with their practice since Debian 10? At the time, the added space was negligible even with only 4 GB of storage.

Reply:
To avoid holding up the rest of this PR, I've split out the question of whether and under what circumstances distro-seed package lists should include documentation packages:

Include libiio0-doc in non-minimal packagelist files #70 

I'll pull this one package out and put it in a separate PR.